bwpy/mea_viewer.py

```diff
@@ -0,0 +1,77 @@
+from . import signal
+import abc
+import plotly.graph_objects as go
+import numpy as np
+from scipy.stats import norm
+
+
+class Viewer(abc.ABC):
+    @abc.abstractmethod
+    def build_view(self, slice, bin_size, view_method):
+        pass
```

Comment:
Nice, but not  required for our purposes as a one-off debug tool :)

Reply:
Should we keep it?  
I thought that maybe in the future it could be useful to have another viewer